tilauskasittely/rahtikirja_pdf.inc

```diff
@@ -320,7 +320,7 @@ for ($tulostuskpl=1; $tulostuskpl<=$tulostakolli; $tulostuskpl++) {
     $pdf->draw_rectangle(696, 60, 684, 300,                      $firstpage, $rectparam);
     $pdf->draw_text(65,  687,  $rakir_row["osoite"],                $firstpage, $kirj);
     $pdf->draw_rectangle(684, 60, 672, 300,                      $firstpage, $rectparam);
-    $pdf->draw_text(65,  675,  $rakir_row["postino"]."  ".$rakir_row["postitp"]."  ".$rakir_row["maa"]."    ".$rakir_row["puhelin"],  $firstpage, $kirj);
+    $pdf->draw_text(65,  675,  $rakir_row["postino"]."  ".$rakir_row["postitp"]."  ".$rakir_row["maa"]."    ".$postirow["toim_puh"],  $firstpage, $kirj);
```

Comment:
@tonyselkamo Onko oikein, että toim_puh haetaan eri muuttujasta kuin muut toimitusositteen tiedot? Jos ok, niin miksi näin?